src/main/java/com/tstepnik/planner/service/StatisticsService.java

```diff
@@ -56,7 +57,7 @@ public Integer countArchivedTasks() {
         return numberOfTasks.intValue();
     }
 
-    public Integer countFinishTasks() {
+    public Integer countFinishedTasks() {
         User loggedUser = authService.getLoggedUser();
         LocalDateTime now = LocalDateTime.now();
         Long numberOfTasks = taskRepository.countAllUserFinishAndArchivedTasks(loggedUser.getId(), now);
```

Comment:
repo method doesnt match up with service method name. What are we getting from repo? finished and archived or just finished? Bad naming introduces confusion.

Reply:
You need to start paying more attention to what your methods are actually doing and naming them accordingly. This method is a great example. It is named `coundFinishedTasks` but its actually counting finished and archived.